RELEASE_NOTES.md

```diff
@@ -0,0 +1,42 @@
+# v5.1.4 libdrizzle redux (2013-04-25)
+
+# Changelog
+
+There are several differences from the main libdrizzle. The following notes are
+from the libdrizzle manual
+
+Differences from Libdrizzle
+---------------------------
+
+- The server-side functionality has been removed, it no longer acts as both a
+  client and server API.
+- The Drizzle prototype library functions have been removed. It now only talks
+  to MySQL compatible servers.
+- API functions have been simplified. In Libdrizzle there was a big confusion
+  over whether the application or library should be doing the allocation and
+  freeing of objects. It is now less ambiguous.
+- New binlog API added. The library can now connect as a slave or mysqlbinlog
+  client and retrieve binlog events.
+
+Differences from libmysqlclient
+-------------------------------
+
+-  API is slightly different
+-  Currently missing server-side prepared statement support and protocol compression
+
+Dependencies
+------------
+
+Dependency | Version
+-----------|---------
+zlib           | v1.x
+openssl or ssl | v1.x
+
+
+Migration Instructions
```

Comment:
> But it might be obsolete if we assume users are starting with v5.1.4

I think that is a reasonable assumption to make and then we maintain migration instructions from there onwards. @leandro-lucarella-sociomantic do you agree?